pgo-scheduler/pgo-scheduler.go

```diff
@@ -170,6 +172,31 @@ func main() {
 		}
 	}
 
+	// If not using the "disabled" namespace operating mode, start a real namespace controller
+	// that is able to resond to namespace events in the Kube cluster.  If using the "disabled"
+	// operating mode, then create a fake client containing all namespaces defined for the install
+	// (i.e. via the NAMESPACE environment variable) and use that to create the namespace
+	// controller.  This allows for namespace and RBAC reconciliation logic to be run in a
+	// consistent manner regardless of the namespace operating mode being utilized.
+	if namespaceOperatingMode != ns.NamespaceOperatingModeDisabled {
+		if err := createAndStartNamespaceController(kubeClient, controllerManager, scheduler,
+			stop); err != nil {
+			log.Error(err)
+			os.Exit(2)
```

Comment:
I'd rather we do a `log.Fatal` which would trigger an `ox.Exit(1)`

Reply:
Agreed, will start using `log.Fatal` instead of of `os.Exit(2)`.